core/node/shared_metrics/src/binary.rs

```diff
@@ -0,0 +1,58 @@
+use async_trait::async_trait;
+use serde::Serialize;
+use vise::{EncodeLabelSet, Info, Metrics};
+
+mod values {
+    use super::BinMetadata;
+    include!(concat!(env!("OUT_DIR"), "/metadata_values.rs"));
+}
+
+pub use values::BIN_METADATA;
+use zksync_health_check::{CheckHealth, Health, HealthStatus};
```

Comment:
- Please don't split `use`s with `mod` declarations; it'll lead to bogus import grouping (like here `zksync_health_check` belongs to external imports).
- Please prepend `self::` for internal imports so that they are easy to distinguish from external ones.